README.md

```diff
@@ -82,14 +78,13 @@ The LINC Brain ecosystem is forked from the [DANDI Archive project](https://gith
 you towards common questions encountered within the LINC Brain project.
 
 * To learn how to interact with the LINC Brain Archive (e.g. a forked DANDI archive) and for examples on how to use the DANDI Client in various use cases,
-see [the handbook](https://www.dandiarchive.org/handbook/)
+see [the DANDI handbook](https://www.dandiarchive.org/handbook/)
   (specifically the sections on using the CLI to
 [download](https://www.dandiarchive.org/handbook/12_download/) and
 [upload](https://www.dandiarchive.org/handbook/13_upload/) `Dandisets`).
 
 * To get help:
-  - ask a question: https://github.com/dandi/helpdesk/discussions
-  - file a feature request or bug report: https://github.com/dandi/helpdesk/issues/new/choose
-  - contact the DANDI team: help@dandiarchive.org
+  - file a feature request or bug report: https://github.com/lincbrain/linc-archive/issues/new
+  - contact the LINC team: kabi@mit.edu or akanzer@mit.edu
 
-* To understand how to contribute to the dandi-cli repository, see the [DEVELOPMENT.md](./DEVELOPMENT.md) file.
+* To understand how to contribute to the linc-cli repository, see the [DEVELOPMENT.md](./DEVELOPMENT.md) file.
```

Comment:
`lincbrain-cli`?

Reply:
Thanks Aaron.  Since the repository is named `linc-cli`, perhaps we should keep this as is?  Alternatively, we could rename the repo to `lincbrain-cli` to be consistent with the PyPI package?